server/auth/cancel.go

```diff
@@ -12,12 +12,20 @@ import (
 	"decred.org/dcrdex/dex/order"
 )
 
+// freeCancelThreshold is the minimum number of epochs a user should wait before
+// placing a cancel order, if they want to avoid penalization. It is set to 2,
+// which means if a user places a cancel order in the same epoch as its limit
+// order, or the next epoch, the user will be penalized. This ensures that the
+// order remains booked for at least one full epoch and one full match cycle.
```

Comment:
`This ensures that the order remains booked` I think should be rephrased in terms of incentivizing this behavior rather than ensuring it.

Reply:
This was sloppy, but I wasn't commenting on the intention of free cancel period, but rather the reason this particular value was chosen. I'm really trying to say `This value is chosen because it is the minimum value such that the order remains booked for at least one full epoch and one full match cycle`.

Will update.